Gemfile

```diff
@@ -44,6 +44,8 @@ gem 'bootsnap', require: false
 
```

Comment:
### Optional
- [ ] It would be great if we can investigate this warning as we can see in the screenshot below. We will have the opportunity to learn more about eager loading
![Screenshot from 2022-10-06 18-58-36](https://user-images.githubusercontent.com/46388113/194397533-73426ddd-c2c1-4821-80f6-640bada31514.png)

Reply:
Thanks for mentioning this problem,I'll  make sure to fix this warning in the upcoming milestone 🙌👍